test/MUIDataTableFilterInline.test.js

```diff
@@ -0,0 +1,330 @@
+import Checkbox from '@material-ui/core/Checkbox';
```

Comment:
I think these tests may be redundant. I'll explain more in the review overview.

Reply:
I'd keep it as separate components, but indeed the tests for them are almost identical, except testing labels & footer. Maybe the tests for both components could be aggregated in one file. What do you think?